tests/features/consultation.feature

```diff
@@ -0,0 +1,78 @@
+@api
+Feature: Consultation content creation
+  In order to have Consultation on the site
+  As an editor
+  I need to be able to create and see Consultation items
+
+  @javascript
+  Scenario: Creation of a Consultation content through the UI.
+    Given I am logged in as a user with the "create oe_consultation content, access content, edit own oe_consultation content, view published skos concept entities, manage corporate content entities" permission
+    And the following document:
+      | name          | file       |
+      | My Document 1 | sample.pdf |
+
+    When I visit "the Consultation creation page"
+    And I fill in "Page title" with "Consultation title"
+    And I fill in "Introduction" with "Introduction text"
+    And I set "Opening" to the date "14-01-2021"
+    And I set "Deadline date" to the date "31-01-2021 00:00" using format "d-m-Y H:i"
+    And I fill in "Departments" with "Associated African States and Madagascar"
+    And I fill in "Target audience" with "Target audience text"
+    And I fill in "Why we are consulting" with "Why we are consulting text"
+    And I fill in "Respond to the consultation" with "Respond to the consultation text"
+    And I fill in "Respond to the consultation (closed status text)" with "Respond to the consultation (closed status text) text"
+    And I fill in "URL" with "http://respond.com"
+    And I fill in "Link text" with "Respond to the questionnaire"
+    And I fill in "Consultation outcome" with "Consultation outcome text"
+    And I fill in "Use existing media" with "My Document 1"
+    And I fill in "Additional information" with "Additional information text"
+    And I fill in "Legal notice" with "Legal notice text"
+
+    # Create General contact.
+    And I press "Add new Contact"
+    And I wait for AJAX to finish
+    And I fill in "Name" with "General contact"
+    And I fill in "Organisation" with "General contact organisation"
```

Comment:
Why do we need to fill optional fields to the Contact entity if we don't check them after? I think we can remove them, or need to check that they exist on the page.